front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/Widgets/_shared/BaseCard.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+import { Box, Image, stylingConsts } from '@citizenlab/cl2-component-library';
+import styled from 'styled-components';
+
+import { CARD_IMAGE_ASPECT_RATIO } from 'api/project_images/useProjectImages';
+
+import ImagePlaceholder from 'components/ProjectCard/ImagePlaceholder';
+
+export const CardContainer = styled(Box)`
+  &:hover {
+    h4 {
+      color: ${({ theme }) => theme.colors.tenantPrimary};
+      text-decoration: underline;
+    }
+
+    .project-image-container > * {
+      transform: scale(1.2);
+    }
+  }
+`;
+
+const ProjectImageContainer = styled.div`
+  width: 100%;
+  display: flex;
+  aspect-ratio: ${CARD_IMAGE_ASPECT_RATIO};
+  margin-right: 10px;
+  overflow: hidden;
+  position: relative;
+  border-radius: ${stylingConsts.borderRadius};
+
+  & > * {
+    transition: transform 0.3s;
+  }
+`;
+
+interface Props {
+  imageUrl?: string;
+}
+
+export const CardImage = ({ imageUrl }: Props) => {
+  return (
+    <Box>
+      <ProjectImageContainer className="project-image-container">
+        {imageUrl ? (
+          <Image
+            src={imageUrl}
+            alt=""
```

Comment:
Same comment as I had earlier - did we end up implementing an alt text for admin publication images in the back office? If so I think we should use that here, and otherwise maybe these images should be hidden for screenreader users?

Reply:
Yeah Edwin recently released something related to this. Needs to be done for all new widgets. Created a separate ticket for it here: https://www.notion.so/govocal/Add-alt-text-to-images-in-all-cards-1499663b7b268004860dd90500aac46c